### ISSUE TITLE
#36379021------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1154]
<strong>Source:</strong>`<span class="title" id="8fde1960-9d51-05a7-e44c-1b007d0c5da6"><slot></slot></span>`

<strong>Element Location: </strong>["calcite-chip[data-name=\"statusCount1\"]","#\\38 fde1960-9d51-05a7-e44c-1b007d0c5da6"]

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix any of the following:
  Element has insufficient color contrast of 2.46 (foreground color: #ffffff, background color: #e8912e, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

<strong>Tags:</strong> cat.color,wcag2aa,wcag143,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/color-contrast?application=axeAPI
